### PR TITLE
applications: nrf_desktop: Common default Bluetooth name and appearance

### PR DESCRIPTION
**Pull request by MarekPieta**
The common default values for Bluetooth name and appearance are defined automatically relying on nRF Desktop application configuration.

Jira: NCSDK-17327

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -218,6 +218,8 @@ nRF Desktop
 
   * Removed separate configurations enabling :ref:`zephyr:shell_api` (:file:`prj_shell.conf`).
     Shell support can be enabled for a given configuration with the single Kconfig option (:ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>`).
+  * nRF Desktop dongles by default use Bluetooth appearance (:kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE)` of Keyboard.
```
[the remark this answers is not in the document]
Comment (annwoj) [reply doc/nrf/releases/release-notes-changelog.rst:221]: TBC: Is there a reason why Keyboard is capitalized?

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -218,6 +218,8 @@ nRF Desktop
...
+    This improves consistency with the used HID boot interface.
```
Comment (annwoj) [inline doc/nrf/releases/release-notes-changelog.rst:222]: Would it be correct to say "this setting improves (...)" ?

**Review by mkapala-nordic — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:222]: Maybe `The new default configuration value improves ... `?

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:221]: In the `Bluetooth Assigned Numbers` document the appearance values seem capitalized so I kept it capitalized here for consistency. Still I think that we could desist from capitalizing it if you prefer (I have no strong opinion on this one). :)

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (annwoj) [reply doc/nrf/releases/release-notes-changelog.rst:222]: Yup, sounds great!

**Review by kapi-no — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (annwoj) [reply doc/nrf/releases/release-notes-changelog.rst:221]: If possible - let's keep it in lowercase. Most of the occurrences are written in lower case, there's no reason to additionally highlight it :) I edited the suggestion

File: applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/prj.conf [hunk 3]
```diff
@@ -70,9 +70,6 @@ CONFIG_USB_NRFX_EVT_QUEUE_SIZE=16
 CONFIG_BT_PRIVACY=y
 CONFIG_BT_LL_SW_SPLIT=y
 
-CONFIG_BT_DEVICE_NAME="Dongle nRF52 Desktop"
-CONFIG_BT_DEVICE_APPEARANCE=962
```
[the remark this answers is not in the document]
Comment (MarekPieta) [reply applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/prj.conf:74]: Yes, it is. The dongle act as HID boot keyboard so this actually improves consistency :)

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:221]: Fixed

**Review by annwoj — APPROVED** [review]

**Pull request merged**